test/app.test.js

```diff
@@ -0,0 +1,279 @@
+
+			await sendData('errorUser', 'errorPass')
+
+			const errorDiv = driver.findElement(By.className('w-form-fail'));
+			const errorMessage = await errorDiv.getText();
+
+			assert.equal(message, errorMessage);
+
+			await driver.sleep(500);
+		})
+
+		it('test for successful login', async function(){
+
+			await sendData(VALID_USER, VALID_PASSWORD);
+
+			const url = await driver.getCurrentUrl();
+			const condition = url.endsWith('/services/');
+
+			assert.ok(condition);
+		})
+
+	})
+
+	describe('test for signup functionality', function() {
+		let email;
+		let username;
+		let password;
+		let confirmPassword;
+		let button;
+
+		before(async function(){
+			await driver.get(SIGNUP_URL);
+			await driver.sleep(500);
+		})
+
+		function sendData(nm, em, p1, p2){
+			username.sendKeys(nm);
+			email.sendKeys(em);
+			password.sendKeys(p1);
+			confirmPassword.sendKeys(p2);
+
+			button.click();
+		}
+
+		beforeEach(async function() {
+			email = driver.findElement(By.id('my-muni-Email address'));
+			username = driver.findElement(By.id('my-muni-Username'));
+			password = driver.findElement(By.id('my-muni-Password'));
+			confirmPassword = driver.findElement(By.id('my-muni-Confirm password'));
+			button = driver.findElement(By.className('button form-submit w-button'));
+
+			username.clear();
+			password.clear();
+			email.clear();
+			confirmPassword.clear();
+		});
+
+		it('check for non-matching password error', async function(){
+
+			sendData('username', 'test@email.com', 'test-pass', 'incorrect')
+
+			await driver.sleep(2000)
+
+			const errorDiv = driver.findElement(By.className('w-form-fail'));
+			const innerDiv = await errorDiv.findElement(By.tagName('div'));
+			const errorMessage = await innerDiv.getText();
+
+			const condition = errorMessage.includes("Passwords don't match");
+
+			assert.ok(condition)
+		})
+
+		it('test for signup success message', async function() {
+
+			sendData('username', 'test@email.com', 'test-pass', 'test-pass')
+
+			await driver.sleep(1500)
+
+			const successDiv = driver.findElement(By.className('w-form-done'));
+			const innerDiv = await successDiv.findElement(By.tagName('div'));
+			const successMessage = await innerDiv.getText();
+
+			const condition = successMessage.includes("Thank you! Your submission has been received!");
+
+			assert.ok(condition)
+		});
+	});
+
+	describe('logout test', function() {
+		before(async function(){
+			await driver.get(LOGIN_URL);
+			await driver.sleep(1000)
+			const usernameBox = driver.findElement(By.id('my-muni-Username'));
+			const passwordBox = driver.findElement(By.id('my-muni-Password'));
+
+			usernameBox.sendKeys(VALID_USER);
+			passwordBox.sendKeys(VALID_PASSWORD);
+			const button = driver.findElement(By.className('button form-submit w-button'));
+			button.click();
+
+			await driver.sleep(2500);
+		});
+
+		it('verify logout worked correctly', async function() {
+			const button = driver.findElement(By.className('icon nav-menu__icon'));
```

Comment:
you need to use something specific to the logout button - can you select by text content?